middlewares/selectAndAuthzInstanceQuestion.sql

```diff
@@ -2,9 +2,18 @@
 WITH instance_questions_info AS (
     SELECT
         iq.id,
-        (lag(iq.id) OVER w) AS prev_instance_question_id,
-        (lead(iq.id) OVER w) AS next_instance_question_id,
-        qo.question_number
+        -- prev and next instance questions are JSON objects to pass directly 
+        -- to partials that render links inside of instance question pages.
+        jsonb_build_object(
+            'id', (lag(iq.id) OVER w)
+        ) AS prev_instance_question,
+        jsonb_build_object(
+            'id', (lead(iq.id) OVER w),
+            'score_perc', (lead(iq.score_perc) OVER w),
```

Comment:
I can't figure out where this variable is used?

Reply:
Nowhere. Removed!